src/worker.js

```diff
@@ -81,23 +62,54 @@ var wasmOffsetData;
 this.onmessage = function(e) {
   try {
     if (e.data.cmd === 'load') { // Preload command that is called once per worker to parse and load the Emscripten code.
+#if MINIMAL_RUNTIME
+#if MODULARIZE_INSTANCE
+      var imports = Module;
+#else
+      var imports = {};
```

Comment:
how does this empty imports object work? is it filled later? (seems not?) do we not need imports somehow?

Reply:
The imports are filled from `makeAsmImportsAccessInPthread()` function.